CHANGELOG.asciidoc

```diff
@@ -88,6 +121,151 @@ https://github.com/elastic/beats/compare/v8.7.1...v8.8.0[View commits]
 
 - Add `event.category` and `event.type` to Sysmon module for EventIDs 8, 9, 19, 20, 27, 28, 255. {pull}35193[35193]
 
+[[release-notes-8.7.1]]
+=== Beats version 8.7.1
+https://github.com/elastic/beats/compare/v8.7.0\...v8.7.1[View commits]
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix Beats started by agent not respecting the allow_older_versions: true configuration flag {issue}34227[34227] {pull}34964[34964]
```

Comment:
This has been released already, see it on Mar 30 towards the bottom of https://github.com/elastic/beats/commits/v8.7.1

I think the one below has also been released already.

I can see this in the 8.7.1 changelog that was forward ported to main which is odd: https://github.com/elastic/beats/pull/35208/files

Maybe it was merged to main after 8.8 was created, in which case we need to backport that PR to main and then regenerate this.

Reply:
What you are mentioning is in the 8.7.1 section, as everything was missing in this branch, I added the content of 8.7.0 and 8.7.1 release here and deleted duplicates in the 8.8.1 section.

Reply:
Ah I was expecting to only see 8.8 relate content, thanks.